### PR TITLE
Detect automatically if toolbox is installed

### PR DESCRIPTION
**Pull request by shaffe-fr**
Hi,
This allows to detect if toolbox for Windows is installed.
Thanks!

### CODE REVIEW

File: PhpStorm Protocol (Win)/run_editor.js [hunk 1]
```diff
@@ -63,6 +63,14 @@ if (match) {
     shell.AppActivate(settings.window_title);
 }
 
+function isToolboxInstalled() {
+	var shell = new ActiveXObject('WScript.Shell'),
+        appDataLocal = shell.ExpandEnvironmentStrings("%localappdata%"),
+        toolboxDirectory = appDataLocal + '\\JetBrains\\Toolbox\\apps\\PhpStorm\\ch-0';
```
Comment (aik099) [inline PhpStorm Protocol (Win)/run_editor.js:69]: Is the toolbox installed into a `ch-0` folder?

Comment (shaffe-fr) [reply PhpStorm Protocol (Win)/run_editor.js:69]: It's the same folder used in the `configureToolboxSettings` function.

Comment (shaffe-fr) [reply PhpStorm Protocol (Win)/run_editor.js:69]: To really answer your question @aik099, Toolbox is installed in %localappdata%\Jetbrains\Toolbox, but this also checks that PHPStorm is installed and that can prevent some errors.

Comment (aik099) [reply PhpStorm Protocol (Win)/run_editor.js:69]: Thank you.

**Comment by aik099** [comment]
Merging, thank you @shaffe-fr .

**Pull request merged**